pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Summary of the PR
+
+*Please summarize here why the changes in this PR are needed.*
+
+### Requirements
+
+Before submitting your PR, please make sure you addressed the following
+requirements:
+
+- All commits in this PR are signed (with `git commit -s`), and follow
+  [the 50/72 git commit rule](https://www.midori-global.com/blog/2018/04/02/git-50-72-rule).
+- All added/changed functionality is tested.
```

Comment:
I would rephrase this to: "All added/changed functionality has a corresponding unit/integration test" to be more explicit. Otherwise people can think this means only that you tested it locally.